doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,730 @@
+
+You will see that one of the columns includes the answers to the test made by people who did not specify their feelings towards religion. Let's delete that column.
+
+.. code-block:: python
+
+   relig_pivot.drop(relig_pivot.columns[0], axis=1, inplace=True)
+
+.. image:: images/workshop-2/religiosity-pivot-table.png
+   :alt: Religiosity pivot table for ANOVA analysis.
+
+Now let's rename the column names (for simplicity) and remove the records that have missing values (with ``dropna``). The argument ``inplace=True`` allows you to directly modify a dataframe without having to create a new one.
+
+.. code-block:: python
+
+   relig_col_names = list(relig_pivot.columns)
+   relig_pivot.columns = ["A", "B", "C", "D"]
+   relig_pivot.dropna(how="all", inplace=True)
+
+.. note::
+
+   In the variable ``religiosity_col_names`` we have stored the names of the values in order. This implies that
+
+   A = Extremely important
+   B = Not at all important
+   C = Not very important
+   D = Somewhat important
+
+To perform ANOVA, we are going to use a Python library called "stats", in particular the f_oneway function. You can find more information about this library by typing ``stats`` in the Help Pane. And if you want to learn more about the function we are going to use, type ``stats.f_oneway`` in the Help pane:
+
+.. image:: images/workshop-2/help-pane-for-stats.f_oneway.png
+   :alt: stats.f_oneway() help in Spyder's Help pane
+
+The help notes that this function requires that you pass, as an array-like, the measurements for each group (in our case, one sample for A, one for B, one for C, and one for D). Since we have empty cells in each column, let's remove them from each column (the f_oneway function does not support missing values).
+
+.. code-block:: python
+
+   relig_samples = [relig_pivot[col].dropna() for col in relig_pivot]
+
+In the above line of code we are creating a list with 4 elements: the test results of each column A, B, C, and D, without the missing values (they are removed with ``dropna()``).
+
+Now, let's run ANOVA and store the output in two variables: ``relig_fvalue`` and ``relig_pvalue``. Notice that since ``relig_samples`` is a list, we must pass the argument with an asterisk (``*relig_samples``).
+
+.. code-block:: python
+
+   relig_fvalue, relig_pvalue = stats.f_oneway(*relig_samples)
+
+With the help of stats.f_oneway in the Help Pane you can read a description of the output of this function, some interesting notes about the test, and some references that you can use to better understand the nature of this analysis.
+
+.. important::
+
+   In the Help component you will read that ANOVA requires certain assumptions to be satisfied. In this workshop we have not checked those assumptions because our goal is merely to show some Spyder functions that make the research work easier. Thus, the results obtained in this example with ANOVA should not be taken too rigorously.
```

Comment:
```suggestion
   In the Help pane you will read that ANOVA requires certain assumptions to be satisfied. In this workshop we have not checked those assumptions because our goal is merely to show some Spyder functions that make the research work easier. Thus, the results obtained in this example with ANOVA should not be taken too rigorously.
```